docs/intro.asciidoc

```diff
@@ -29,33 +28,29 @@ NOTE: The minimum required version of the APM Server is 6.5.0
 === Additional Components
 
 APM Agents work in conjunction with the {apm-server-ref-v}/index.html[APM Server], {ref}/index.html[Elasticsearch], and {kibana-ref}/index.html[Kibana].
-Please view the {apm-overview-ref-v}/index.html[APM Overview] for details on how these components work together. 
+Please view the {apm-overview-ref-v}/index.html[APM Overview] for details on how these components work together.
 
 [float]
 [[get-started]]
-=== Get started
```

Comment:
It might be an option to stick with "Get Started" as the title here. We could then change the name of the standalone "setup" page to "get started". That might help to differentiate between the "setup/get started" section and the "application-server-setup" section?

Reply:
Do you think this would still make sense when inlining the application-server-setup page into the `-javaagent` setup?